benchmarks/haskell/Benchmarks.hs

```diff
@@ -1,5 +1,6 @@
 {-# LANGUAGE CPP #-}
 {-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE NumericUnderscores #-}
```

Comment:
This fails with old GHCs.

Reply:
I did not realize how new that extension is.